server/src/domain/services/job-service.js

```diff
@@ -17,16 +17,21 @@ function _isKindOfProjectWantedOnJobBoard(project) {
   return project.kind === 'cost_reimbursable' || project.kind === 'fixed_price';
 }
 
-function _filterProjectByStatusAndKind(projects) {
+function _doesProjectNotBelongToOcac(project) {
```

Comment:
Ta convention était d'utiliser "is" dans le nom.
Ce ne serait pas mieux "_isProjectNotOwnedByOcac" ?